src/Components/ServiceScene/Breakdowns/SelectLabelActionScene.tsx

```diff
@@ -110,6 +261,23 @@ export class SelectLabelActionScene extends SceneObjectBase<SelectLabelActionSce
     );
   }
 
+  public onClickNumericFilter = (variableType: VariableFilterType) => {
+    const detectedFieldFrame = getDetectedFieldsFrame(this);
+    const fieldType = getDetectedFieldType(this.state.labelName, detectedFieldFrame);
+
+    // @todo, what can we do if we can't determine the field type?
```

Comment:
What are the chances of this happening?

Reply:
I would hope very low, should only happen with an unsupported or broken Loki that isn't returning the field types in the detected_field response. I believe my concern was that the detected_fields response isn't there, but if it's not there we wouldn't have any field panels to render this component on 🤷